pyvista/core/objects.py

```diff
@@ -47,10 +47,18 @@ def __init__(self, *args, **kwargs):
             else:
                 raise TypeError(f'Table unable to be made from ({type(args[0])})')
 
+    @staticmethod
+    def _prepare_arrays(arrays):
+        arrays = np.array(arrays, copy=False)
```

Comment:
If this is meant to say "turn it into an array unless it's already an array", I'd prefer
```suggestion
        arrays = np.asarray(arrays)
```